docs/src/developer-guide/working-with-custom-formatters.md

```diff
@@ -132,13 +132,18 @@ Each `message` object contains information about the ESLint rule that was trigge
 The formatter function receives an object as the second argument. The object has two properties:
```

Comment:
```diff
- The object has two properties:
+ The object has two or three properties:
```

or maybe just "the following properties"?

Reply:
Thanks for catching this! Fixed in c772b6c.